dashboard/src/actions/authActions.js

```diff
@@ -28,16 +28,15 @@ export function waitForKeycloak(getState) {
       setTimeout(check, 250, resolve, reject);
     }
   }
-  return new Promise((resolve, reject) => {
-    check(resolve, reject);
-  });
+  return new Promise((resolve, reject) => check(resolve, reject));
 }
 
 // Perform some house keeping when the user logs in
 export const authCookies = () => async (dispatch, getState) => {
-  await waitForKeycloak(getState);
+  await waitForEndpoints(getState);
   const keycloak = getState().apiEndpoint.keycloak;
   if (keycloak.authenticated) {
+    // Set the cookie expiry to milliseconds since the UNIX Epoch
```

Comment:
This is not a very clear comment. You're apparently converting the *units* of the OIDC token expiration from seconds to milliseconds, not "setting the [expiration] to [how many?] milliseconds..."